File: 4.cpp
#include <stdio.h>
#include <math.h>
#include <iostream>
using namespace std;
int check(int n)
{
    if (n == 1)
    {
        return 0;
    }
    for (int i = 2; i <= sqrt(n); i++)
    {
        if (n % i == 0)
        {
            return 0;
        }
    }
    return 1;
}
int main()
{
    int a, b;
    printf("nhap a:");
    scanf("%d", &a);
    cout << "nhap b:";
    cin >> b;
    cout << "(";
    for (a; a < b; a++)
    {
        if (check(a) == 1)
        {
            cout << a << " ";
        }
    }
    cout << ")";
}

File: python/4.py
import math


def check(n):
    if n < 2:
        return False
    for i in range(2, int(math.sqrt(n))+1):
        if n % i == 0:
            return False
    return True


a = int(input("nhap A: "))
b = int(input("nhap B: "))
print("cac so nguyen to: ")
for i in range(a, b+1):
    if check(i):
        print(i)
